app/src/main/java/com/github/fribourgsdp/radio/UserProfileActivity.kt

```diff
@@ -112,12 +105,12 @@ class UserProfileActivity : AppCompatActivity() {
         firebaseAuth = FirebaseAuth.getInstance()
         usernameField = findViewById(R.id.username)
         launchSpotifyButton = findViewById(R.id.launchSpotifyButton)
-        logoutButton = findViewById(R.id.logoutButton)
+        logoutButton = findViewById(R.id.homeButton)
```

Comment:
Here, you renamed the ID but not the variable.

Reply:
Done